aspnet/tutorials/first-mvc-app/adding-view.rst

```diff
@@ -106,35 +105,34 @@ Controllers are responsible for providing whatever data or objects are required
 Currently, the ``Welcome`` method in the ``HelloWorldController`` class takes a ``name`` and a ``ID`` parameter and then outputs the values directly to the browser. Rather than have the controller render this response as a string, let’s change the controller to use a view template instead. The view template will generate a dynamic response, which means that you need to pass appropriate bits of data from the controller to the view in order to generate the response. You can do this by having the controller put the dynamic data (parameters) that the view template needs in a ``ViewData`` dictionary that the view template can then access.
 
 Return to the *HelloWorldController.cs* file and change the ``Welcome`` method to add a ``Message`` and ``NumTimes`` value to the ``ViewData`` dictionary. The ``ViewData`` dictionary is a dynamic object, which means you can put whatever you want in to it; the ``ViewData`` object has no defined properties until you put something inside it. The `MVC model binding system <http://docs.asp.net/projects/mvc/en/latest/models/index.html>`__ automatically maps the named parameters (``name`` and ``numTimes``) from the query string in the address bar to parameters in your method. The complete *HelloWorldController.cs* file looks like this:
-
+	
 .. literalinclude:: start-mvc/sample/src/MvcMovie/Controllers/HelloWorldController.cs
-  :language: c#
-  :lines: 152-171
+	:language: c#
+	:lines: 152-172
 
 The ``ViewData`` dictionary object contains data that will be passed to the view. Next, you need a Welcome view template.
 
 - Right click on the *Views/HelloWorld* folder, and then **Add > New Item**.
 - In the **Add New Item - MvcMovie** dialog
 
-  - In the search box in the upper-right, enter *view*
-  - Tap **MVC View Page**
-  - In the **Name** box, enter *Welcome.cshtml*
-  - Tap **Add**
+	- In the search box in the upper-right, enter *view*
+	- Tap **MVC View Page**
+	- In the **Name** box, enter *Welcome.cshtml*
+	- Tap **Add**
 
 You'll create a loop in the *Welcome.cshtml* view template that displays "Hello" ``NumTimes``. Replace the contents of *Views/HelloWorld/Welcome.cshtml* with the following:
 
 .. literalinclude:: start-mvc/sample/src/MvcMovie/Views/HelloWorld/Welcome.cshtml
-  :language: HTML
+   :language: HTML
 
 Save your changes and browse to the following URL:
 
-  \http://localhost:xxxx/HelloWorld/Welcome?name=Rick&numtimes=4
+http://localhost:xxxx/HelloWorld/Welcome?name=Rick&numtimes=4
```

Comment:
Either escape this URL with a backslash or make it preformatted text with double back ticks.